<launch>
    <node pkg="payload_estimation" type="gp_prediction.py" name="gp_test_node" output="screen">
        <param name="test_csv" value="/home/robat/catkin_ws/src/algorithmic_payload_estimation/payload_estimation/data/processed/csv/default_test_data.csv" />
        <param name="model_filename" value="/home/robat/catkin_ws/src/algorithmic_payload_estimation/payload_estimation/gp_models/gp_model.pkl" />
        <param name="results_csv" value="/home/robat/catkin_ws/src/algorithmic_payload_estimation/payload_estimation/data/result/results.csv" />
    </node>
</launch>